icinga2/postgresql-server.sls

```diff
@@ -1,4 +1,6 @@
-{% if salt['pillar.get']('icinga2:postgres:use_formula', False) %}
+{% from "icinga2/map.jinja" import icinga2 with context %}
+
+{% if icinga2.get('postgres:use_formula', False) %}
```

Comment:
I've run this against the new testing setup and this change doesn't work (the pillar value is set to use the formula):

* https://travis-ci.org/myii/icinga2-formula/builds/623587122
* https://travis-ci.org/myii/icinga2-formula/jobs/623587127#L1268-L1277 -- this shows how this state gets rendered

The reason for this is due to the current structure of `map.jinja`, since only the `lookup` from the pillar is merged into the map:

https://github.com/saltstack-formulas/icinga2-formula/blob/d0261f65958796eb8f884cb2ee2b39c53d124d43/icinga2/map.jinja#L13

I.e. This `if` will never run, only the `else`.

If this PR is going to proceed, the correct resolution would be to merge in the rest of the pillar, as we do in many of our other formulas, such as the [`template-formula`](https://github.com/saltstack-formulas/template-formula/blob/341f495336da0e35d92b3b4acda30f9efa44ec52/template/map.jinja#L31-L47).